index.js

```diff
@@ -0,0 +1,51 @@
+const GIPHY_URL = 'https://api.giphy.com/v1/stickers/search?'; 
+const API_KEY = 'api_key=E26iZ0BIEszi7RA4GeEmXzJPRpkSeQ6m';
+const SEARCH_PARAM = '&q=';
+const AMOUNT = '&limit=50';
+
+let numberElements = 5;
+let userWord;
+let url;
+let json;
+
+const input = document.getElementById('findInput');
+const imgWrap = document.querySelector('.wrap-for-copy').cloneNode(true);
+
+document.querySelector('.wrap-for-copy').remove();
+imgWrap.classList.remove('wrap-for-copy');
+
+const replaceImage = () => {
+  //shuffle array-------------
+  let array = [...json.data];
+  array.sort(function() {
+      return Math.random() - 0.5
+  });
+  //--------------------------
+  const needGifArr = array.slice(0, numberElements)
+  needGifArr.forEach(gif => {
+    const newImg = imgWrap.cloneNode(true);
+    newImg.querySelector('img').src = gif.images.original.url;
+    newImg.querySelector('img').alt = gif.title;
+    document.getElementById('listImages').appendChild(newImg);
```

Comment:
this expensive operation. It should gather all elements firstly and then append all of them to the dom tree